examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -152,6 +160,11 @@
   ],
   "parameters": [
     {
+      "name": "DEPLOYMENT_LIMIT_SIZE",
```

Comment:
Why not POD_MEMORY or something else that makes it clear
1) what the limit is applied to (pods)
2) what units/resource is being limited (memory)

i don't love pod_memory itself, but i can't think of a more accurate description that sounds good.